packages/flutter/lib/src/material/tabs.dart

```diff
@@ -1308,14 +1309,28 @@ class _TabBarState extends State<TabBar> {
 
     if (widget.isScrollable) {
       _scrollController ??= _TabBarScrollController(this);
-      tabBar = SingleChildScrollView(
-        dragStartBehavior: widget.dragStartBehavior,
-        scrollDirection: Axis.horizontal,
-        controller: _scrollController,
-        padding: widget.padding,
-        physics: widget.physics,
-        child: tabBar,
-      );
+      if (Platform.isAndroid && widget.physics == null) {
```

Comment:
This should probably just always apply when it is scrollable, not just on Android. Any platform could be applying a glowing overscroll indicator. Also, rather than the user setting widget.physics as a signal, we should probably just add the ability for the user to provide an overriding scroll behavior.

There are a few widgets where we manipulate the the inherited scroll behavior already, here is an example:
https://github.com/flutter/flutter/blob/4914a4090c0a3e3973200e966733583a60b5674e/packages/flutter/lib/src/widgets/list_wheel_scroll_view.dart#L792

Here it is toggling the scrollbar off, instead you could set `overscroll: false`.